cmd/context/update-kubeconfig.go

```diff
@@ -62,10 +63,16 @@ func ExecuteUpdateKubeconfig(okCtx *okteto.OktetoContext, kubeconfigPaths []stri
 	if err != nil {
 		return err
 	}
-	if err := okClient.Kubetoken().CheckService(okCtx.Name, okCtx.Namespace); err == nil {
-		updateCfgAuthInfoWithExec(okCtx)
+
+	if !utils.LoadBoolean(model.OktetoUseStaticKubetokenEnvVar) {
```

Comment:
I would switch this to be. I think it's clearer to read the positive branch first:
```suggestion
	if utils.LoadBoolean(model.OktetoUseStaticKubetokenEnvVar) {
```

Reply:
good call, done ✅